app/presenters/result_set_presenter.rb

```diff
@@ -78,6 +90,22 @@ def signup_links
 
   attr_reader :view_context
 
+  def no_top_results(documents)
```

Comment:
The Rails convention for methods that return true/false is add a `?` to the end of the name. 

Also, I think for readability, this function should have a "truthy" name, i.e. `highlight_top_result?`.
It would make the code easier to read too:

e.g. 
```
documents.length > 2 && 
sort_option && 
sort_option["key"].eq?("-relevance") &&
finder.slug == "/find-eu-exit-guidance-business"
```